src/site.rs

```diff
@@ -346,7 +340,6 @@ impl Site {
 
     /// Builds the site to the `public` directory after deleting it
     pub fn build(&self) -> Result<()> {
-        self.clean()?;
```

Comment:
If a keep this line all files on _public_ directory is deleted. I want to keep these files and copy only files that have been modified on static directory.